### PR TITLE
The continuous timeout lock member should be declared final

### PR DESCRIPTION
**Pull request by eabbott**
The member field created a PR or two ago should have been declared final.

@sit @axiak @HiJon89 @jdwyah

### CODE REVIEW

**Comment by sit** [comment]
:+1: I see no attempts at modification of this, anyway.

**Comment by jhaber** [comment]
lgtm

File: src/main/java/net/spy/memcached/protocol/TCPMemcachedNodeImpl.java [hunk 1]
```diff
@@ -45,7 +45,7 @@
 	private long defaultOpTimeout;
 
 	// operation Future.get timeout counter
```
Comment (mjball) [inline src/main/java/net/spy/memcached/protocol/TCPMemcachedNodeImpl.java:47]: This comment is wrong now :panda_face:

**Comment by mjball** [comment]
:shipit: of course

File: src/main/java/net/spy/memcached/protocol/TCPMemcachedNodeImpl.java (hunk 1 above)
Comment (eabbott) [inline src/main/java/net/spy/memcached/protocol/TCPMemcachedNodeImpl.java:47]: Comment is changed in the other pr

**Pull request merged**